devguide/examples/map-editFeature.js

```diff
@@ -1,68 +1,55 @@
+var overlay1 = new emp3.api.Overlay({
```

Comment:
Agree, would like to save that until next round of edits on the examples so this does not get held out